cloud-sdk/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:latest
```

Comment:
Do we need latest?. `slim` might suffice what we require?

Reply:
Also, is there a GAR version of this image?.

Reply:
I think this is because we had on the card: `test a deployment with the latest cloud-sdk (upgrade from slim) image`, so we thought it stays as latest, do you want me to revert it to slim? Also, I can't find the GAR version of this image, looks like it's only GCR atm.

Reply:
We should use the smallest image possible. So if slim is enough then, we should use that. I doubt `alpine` will be enough, but could try.

Reply:
I've changed it back to slim and will test alpine.

Reply:
Should be rename `eq-app-deploy-image` to ` eq-runner-deploy-image`? I think that's all its used for now. I guess the (Github) repo itself is still named ok but the (colud-sdk) image could now be Runner specific.

Reply:
Regarding `alpine`, it broke as `apt-get` not present so I revert it back to `slim`.

Reply:
I think we use it for `create`, `delete` and `read` secrets tasks in pipelines as well.

Reply:
Ok, re: the name that's fine. It was just an observation given we'd introduced a specific named image for benchmark and then have a generic app named image. It's likely to raise a question in the future.

Reply:
We could rename to be runner specific, then we could just use the cloud-sdk images directly for the secrets tasks. I don't think we necessarily need to avoid using those images directly. For apps it makes the most sense to have a controlled scanned images. For point in time tasks, the risk is minimal. So this repo could stay name as ie, but generates 2 app specific images.

Reply:
Yes, the (GH) repo name is fine. It was the runner deploy image that was now vague. We can raise this separately if necessary and leave it for this card.